### PR TITLE
Portfolio Mobile Menu Hamburger

### PR DESCRIPTION
**Pull request by PraveenKumar141095**
#**In this project milestone**

**I successfully effect the hamburger button for listing mobile menu when toggled and closed when click the X icon
I successfully linked the mobile menu options to its pages**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+});
+
+cancel.addEventListener('click', () => {
+  overlay.style.display = 'none';
+});
+
+
+port.addEventListener('click', () => {
+  mainMenu.style.backgroundColor = 'none';
+});
```
Comment (Santiago220991) [inline script.js:20]: Kindly make sure to fix this issue. :eyes: :eyes:. A project without errors will show to the recruiters your proficiency in coding.
image

File: script.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+const port = document.querySelector('.portfolio');
+
+hamburg.addEventListener('click', () => {
+  mainMenu.style.display = 'flex';
+  overlay.style.display = 'block';
+});
+
+cancel.addEventListener('click', () => {
+  overlay.style.display = 'none';
+});
```
Comment (Santiago220991) [inline script.js:15]: Please make sure that when you click on any option in your hamburger menu the user is redirected to that specific section of your portfolio and the hamburger menu closes. This feature currently is working just for the contact option. :+1: :smiley: :smiley:

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @Praveen1995-10-14 @edahigure,
I'm Santiago your code reviewer for this PR. :wave: :wave:  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
:heavy_check_mark: No linter errors.
:heavy_check_mark: Followed GitHub flow.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
[OPTIONAL]  Kindly make sure that your navbar in the desktop version is rendering correctly. :pray: :pray:   The navbar is rendering incomplete check the image below please.

image

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Praveen1995-10-14👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**